### PR TITLE
Adds test for repo files

### LINKED ISSUE #58: Add unittest to check for specific repo files
This repo should have the following list of files included:
- ./Docker or docker/Docker
- ./docker-compose.yml or ./docker/docker-compose.yml
- ./.env_sample
- ./.gitignore
- ./.travis.yml
- ./.codeclimate.yml
- ./CHANGELOG.md
- ./CODE_OF_CONDUCT.md
- ./CONTRIBUTING.md
- ./.github/ISSUE_TEMPLATE
- ./LICENSE.md
- ./.github/PULL_REQUEST_TEMPLATE
- ./README.md
- ./TROUBLESHOOTING.md
- ./USAGE.md
- ./USE_CASES.md

This PR is only asking for tests of the existence of these files, if the files do not exist when you run the tests - do not worry about the tests not passing. We will identify this and create a new PR for the issue.
Thank you!

### PR DESCRIPTION
**Pull request by cheukyin699**
This should resolve #58. Does not pass tests because of missing files.

### CODE REVIEW

File: tests/test_repofiles.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        ['./TROUBLESHOOTING.md'],
+        ['./USAGE.md'],
+        ['./USE_CASES.md']
+    ]
+
+    def _any_file(self, files):
+        return any(map(path.isfile, files))
+
+    def test_file_existance(self):
+        self.assertTrue(all(map(self._any_file, self.FILES)))
```
Comment (mbernier) [inline tests/test_repofiles.py]: Can you please have this report an error about which file was failing when it fails? Right now, we just know THAT it failed, not why.

**Comment by thinkingserious** [comment]
@cheukyin699

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

**Comment by thinkingserious** [comment]
Hello @cheukyin699,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/30min) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request merged**